app/views/friendships/_friendships.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="border p-3 mb-3 rounded">
+<% @friendships.each do |friendship| %>
+  <tr>
+  <%= friendship %>
+  </tr>
```

Comment:
When this is corrected, your iteration should a ruby object, kindly search for the user who has send this request and display their name instead.